week2/materials/homework/assignments.md

```diff
@@ -4,38 +4,38 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/g3-week2-tithi?region=us-east-1&tab=objects`
```

Comment:
Could you add the link to your website instead of the bucket? 👍

Reply:
Oh ok. I just added both now :)